Gopkg.toml

```diff
@@ -9,7 +9,7 @@ required = [
 
 [[constraint]]
   name = "github.com/linkerd/linkerd2-proxy-api"
-  version = "v0.1.2"
+  revision = "9f5c62278bc10b7bf9bb9facba8ea46b904bc0e8"
```

Comment:
Since this commit is already on master, my preference is to create a v0.1.3 tag in the linkerd2-proxy-api repo, and pin the version to that tag.